accern/default_client.py

```diff
@@ -230,3 +249,22 @@ def handle_request_error(e):
                 err += " with no error message"
         msg = textwrap.fill(msg) + "\n\n(Network error: %s)" % (err,)
         raise error.APIConnectionError(msg)
+def quant_filter(json_data,filters):
+    signals=json_data['signals']
```

Comment:
the function should not which field it is taking values from. move the signals field value to the parent function and change the json_data => raw_data